VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -149,7 +149,6 @@ public void onException(Exception ex) {
                 buttonAddComment.setEnabled(true);
             }
         };
-        createCommentTask.setTaskProcessCallback(null);
```

Comment:
It might make sense to do this automatically from the `setProgressDialog()` implementation when the parameter is a non-null value.